mbed-edge-examples/rpm/mbed-edge-examples.spec

```diff
@@ -1,4 +1,4 @@
-%global forgeurl https://github.com/ARMmbed/mbed-edge-examples
+%global forgeurl https://github.com/PelionIoT/mbed-edge-examples
 %global tag     0.10.0
```

Comment:
This one is really badly behind, 0.10.0... 
But, I agree - better to bump domains ONLY 1st and after that do the version upgrades.